.github/workflows/build-manywheel-images.yml

```
@@ -1,6 +1,9 @@
 name: Build manywheel docker images
 
 on:
+  push:
```

do you need to specify `paths` here too?
```suggestion
  push:
    paths:
      - manywheel/Dockerfile
```

---

oh good call